metrics/README.md

```diff
@@ -0,0 +1,53 @@
+# Metrics Plugins
+
+Fluentd has nine \(9\) types of plugins:
+
+* [Input](../input/)
+* [Parser](../parser/)
+* [Filter](../filter/)
+* [Output](../output/)
+* [Formatter](../formatter/)
+* [Storage](../storage)
+* [Service Discovery](../service_discovery/)
+* [Buffer](../buffer/)
+* [Metrics](./)
+
+This article gives an overview of Metrics Plugin.
+
+## Overview
+
+Sometimes, the input/filter/output plugin needs to save its internal metrics in memory, influxdb or prometheus format ready in instances. Fluentd has a pluggable system called Metrics that lets a plugin store and reuse its internal state as metrics instances.
+
+## How To Use
+
+On Fluentd core, metrics plugin will handled on `<metrics>` on `<system>` to set up easily.
+
+Here is an example with `metrics_local`:
+
+```text
+<system>
+  <metrics>
+    @type local
+  </metrics>
+</system>
+```
+
+## List of Built-in Metrics
```

Comment:
Please add `Plugins`

```diff
- ## List of Built-in Metrics
+ ## List of Built-in Metrics Plugins
```

Reply:
Fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/347/commits/f57d9aa2fac954406f6477338dbb85c7a086b82f.